libs/tsconfig/project.json

```diff
@@ -1,16 +1,22 @@
 {
 	"$schema": "../../node_modules/nx/schemas/project-schema.json",
-	"sourceRoot": "libs/tsconfig/src",
+	"sourceRoot": "libs/tsconfig",
 	"projectType": "library",
 	"targets": {
 		"build": {
-			"executor": "@nrwl/js:tsc",
+			"executor": "@nrwl/web:rollup",
 			"outputs": ["{options.outputPath}"],
 			"options": {
+				"project": "libs/tsconfig/package.json",
+				"entryFile": "libs/tsconfig/src/index.ts",
 				"outputPath": "dist/libs/tsconfig",
-				"main": "libs/tsconfig/src/index.ts",
 				"tsConfig": "libs/tsconfig/tsconfig.lib.json",
-				"assets": ["libs/tsconfig/*.md", "libs/tsconfig/tsconfig.json"]
+				"assets": [
+					"libs/tsconfig/readme.md",
+					"libs/tsconfig/CHANGELOG.md",
+					"libs/tsconfig/tsconfig.json"
+				],
+				"format": ["esm", "cjs"]
 			}
```

Comment:
I see that the rollup executor uses babel by default, but can accept either babel or swc as compiler - we could change it to swc with the option `"compiler": "swc"` and that should be faster - and would presumably negate the need for the babel config file.

Reply:
Started looking at that earlier - there are advantages (build time) and disadvantages (doesn't export types) so we'd need to weigh those up: https://github.com/guardian/csnx/pull/32

Reply:
I don't think having a babel config file is inherently a bad thing though unless you have a specific reason for thinking that?